Classes/common/CDTMutableDocumentRevision.m

```diff
@@ -50,25 +49,14 @@ - (instancetype)initWithDocumentId:(NSString*) documentId
     
     if (self){
         _docId = documentId;
-        _private_body = body ? body : [NSMutableDictionary dictionary];
-        _private_attachments = attachments ? attachments : [NSMutableDictionary dictionary];
+        _body = body ? body : [NSMutableDictionary dictionary];
+        _attachments = attachments ? attachments : [NSMutableDictionary dictionary];
         _sourceRevId = sourceRevId;
     }
     
     return self;
 }
 
-- (void)setBody:(NSDictionary *)body { self.private_body = [body mutableCopy]; }
-
-- (NSMutableDictionary *)body { return self.private_body; }
-
-- (NSMutableDictionary *)attachments { return self.private_attachments; }
-
-- (void)setAttachments:(NSMutableDictionary *)attachments
-{
-    self.private_attachments = [attachments mutableCopy];
```

Comment:
We could but then it would make it harder for someone to use. Instead of 
```objc
CDTMutableDocumentRevision *rev = [CDTMutableDocumentRevision revision];
rev.attachments["myAttachment"] = attachment;
```
we would be forcing a use to 
```objc
CDTMutableDocumentRevision *rev = [CDTMutableDocumentRevision revision];
NSMutableDictionary *attachments = [rev.attachments mutableCopy];
attachments["myAttachment"] = attachment;
rev.attachments
```
 I think the former is a better solution, it keeps the current syntax (no breaking changes), and it is still easy to use.